database/factories/VideoFactory.php

```diff
@@ -21,7 +21,7 @@
         'title' => $faker->sentence($nbWords = 5, $variableNbWords = true),
         'description' => $faker->sentence($nbWords = 10, $variableNbWords = true),
         'category_id' => $categories->random()->id,
-        'reviews' => $faker->numberBetween($min = 0, $max = 5),
+        // 'reviews' => $faker->numberBetween($min = 0, $max = 5),
```

Comment:
If this is no longer needed, please just remove the line. Don't leave "dead code" hanging around in the form of commented out code, this will get confusing in the long run.